pkg/client/repo/oci/oci.go

```diff
@@ -103,40 +104,35 @@ func (r *Repo) List() ([]string, error) {
 }
 
 // getTagManifest returns the manifests of a published tag
-func (r *Repo) getTagManifest(name, version string) (*ocispec.Manifest, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), getTimeout)
-	defer cancel()
-
+func (r *Repo) getTagManifest(chartName, version string) (*ocispec.Manifest, error) {
 	u := *r.url
-	// Form API endpoint URL from repo url
-	u.Path = path.Join("v2", u.Path, name, "manifests", version)
-	req, err := http.NewRequestWithContext(ctx, "GET", u.String(), nil)
-	req.Header.Set("Accept", ImageManifestMediaType)
+	u.Path = path.Join(u.Path, "/", chartName)
```

Comment:
This doesn't seem to make sense as `path.Join` uses forward slashes as separators: "[Join joins any number of path elements into a single path, separating them with slashes](https://pkg.go.dev/path#Join)".

Reply:
Yes but this is to ensure a `/` is prepended before the path to do an easy concat later `u.Host + u.Path`

Example to illustrate the problem:
```
url = "https://localhost:8080"
u.Path = path.Join("", "/", "testchartName")
u.Path == "/testchartName"
```
Without this:
```
url = "https://localhost:8080"
u.Path = path.Join("", "testchartName")
u.Path == "testchartName"
```
And that will break the concat, since the slash is missing. For when the slash is already present when the url is set like this `https://localhost:8080/``path.Join` will normalize the slashes.

The root of this problem is the `u.Host + u.Path`. I need to get the host + path without the schema. I personally found this more readable then this:

```golang
fmt.Sprintf("%s:%s", path.Join(strings.TrimPrefix(r.url.String(), fmt.Sprintf("%s://", r.url.Scheme)), chartName), version)
```

Reply:
Thanks by the way for peer reviewing this PR! Very helpful!